src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -520,6 +520,51 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_bias_bprop()
     this->add_matcher(m, callback);
 }
 
+void ngraph::runtime::cpu::pass::CPUPreFusion::construct_maxpool_relu_switch()
```

Comment:
Can we make this more general ? Templatize it to take two ops types to reverse.

Reply:
If we want to make this more general, then we cannot create new op in the callback because different ops require different arguments. We need to keep both ops and change the inputs/outputs of these four nodes: node_before->op1->op2->node_after.